src/System/PosixCompat/Files.hsc

```diff
@@ -120,7 +120,7 @@ import Control.Exception (bracket)
 import Control.Monad (liftM, liftM2)
 import Data.Bits ((.|.), (.&.))
 import Prelude hiding (read)
-import System.Directory
+import System.Directory hiding (isSymbolicLink)
```

Comment:
Sorry this won't be ok because it will break on directory < 1.2.6 where the function doesn't exist, I have a fix up on a branch which uses explicit imports instead, just testing it now